base/markdown/parse/util.jl

```diff
@@ -93,7 +93,7 @@ function startswith{T<:String}(stream::IO, ss::Vector{T}; kws...)
 end
 
 function startswith(stream::IO, r::Regex; eat = true, padding = false)
-  @assert beginswith(r.pattern, "^")
+  @assert Base.startswith(r.pattern, "^")
```

Comment:
Yes, that's a fair point. I guess I could change it to `beginswith` ;) Either way I'll add it to my list of tasks when I updating Markdown.jl